src/dotty/tools/dotc/core/tasty/TastyFormat.scala

```diff
@@ -485,4 +485,15 @@ object TastyFormat {
     case PRIVATEqualified => "PRIVATEqualified"
     case PROTECTEDqualified => "PROTECTEDqualified"
   }
+
+  /** @return If non-negative, the number of leading references of a length/trees entry.
+   *          If negative, minus the number of leading non-reference trees.
+   */
+  def numRefs(tag: Int) = tag match {
```

Comment:
I think it is essentially an information about the format. So the place looks right to me. Otherwise put, if we change the format of some entries, this method needs to be updated in sync. With the information provided by it, we enable generic traversals of Tasty trees in other (future) modules as well.